update_compilers/install_libraries.sh

```diff
@@ -241,3 +241,64 @@ install_lua() {
 }
 
 install_lua v5.3.5 v5.4.0
+
+install_nsimd() {
+    for VERSION in "$@"; do
+        local DEST=${OPT}/libs/nsimd/${VERSION}
+        if [[ ! -d ${DEST} ]]; then
+            rm -rf /tmp/nsimd
+            mkdir -p /tmp/nsimd
+            pushd /tmp/nsimd
+
+            git clone -b ${VERSION} https://github.com/agenium-scale/nsimd.git
```

Comment:
this hangs on the admin box asking for keys etc

```
+ cd /tmp/nsimd/nsimd/scripts/..
+ git clone git@github.com:agenium-scale/nstools.git
Cloning into 'nstools'...
The authenticity of host 'github.com (140.82.113.3)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
```

Will fix...

Reply:
that seems unrelated to your change...maybe github's SHA changed? :)

Reply:
+ git clone git@github.com:agenium-scale/nstools.git
Cloning into 'nstools'...
The authenticity of host 'github.com (140.82.112.3)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added 'github.com,140.82.112.3' (RSA) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.


ah...we don't clone from a github account. will change to use https://

Reply:
er? No, that's not part of our code here. Is it `scripts/setup.sh` that's doing this?

Reply:
Ah.. It comes from this:

```
NSTOOLS_URL="git@github.com:agenium-scale/nstools.git"
NSTOOLS_URL2="https://github.com/agenium-scale/nstools.git"

###############################################################################
# Build nsconfig (if not already built)

[ -e "${NSTOOLS_DIR}/README.md" ] && \
    ( cd "${NSTOOLS_DIR}" && git pull ) || \
    ( cd "${PWD}/.." && \
      ( git clone "${NSTOOLS_URL}" || git clone "${NSTOOLS_URL2}" ) )
```

and so the error is skipped and the other, non-git URL is tried. So this probably works as-is.